src/coreclr/vm/ClrEtwAll.man

```diff
@@ -3696,6 +3720,15 @@
                            task="Contention"
                            symbol="ContentionLockCreated" message="$(string.RuntimePublisher.ContentionLockCreatedEventMessage)"/>
 
+                    <event value="301" version="1" level="win:Informational"  template="WaitStart_V1"
```

Comment:
It seems like event versions start at 0 with no suffix in the name. I've started this event at 1 and added a prefix, tell me if I need to change that

Reply:
Yeah 0 and no suffix is prefereable. version 1+ is used for making changes to pre-existing events.